### PR TITLE
create user 

### PR DESCRIPTION
**Pull request by sanglevinh**

### CODE REVIEW

File: frontend/src/pages/AccountManager/index.tsx [hunk 1]
```diff
@@ -94,18 +306,67 @@ const AccountManager = () => {
...
+        break;
+      case "DATA_MANAGER":
+        newRole = 10;
+        break;
+      case "OPERATOR":
+        newRole = 20;
+        break;
+      case "GUEST_OPERATOR":
+        newRole = 30;
+        break;
```
Comment (ReiHashimoto) [inline frontend/src/pages/AccountManager/index.tsx:348]: please use enum `ROLE`

File: frontend/src/pages/AccountManager/index.tsx [hunk 2]
```diff
@@ -1,17 +1,226 @@
...
+  name: '',
+  confirmPassword: '',
+}
+
+const ModalComponent =
+   ({
+     onSubmitEdit,
+     setOpenModal,
+     dataEdit,
+   }: ModalComponentProps) => {
```
Comment (ReiHashimoto) [inline frontend/src/pages/AccountManager/index.tsx:43]: just styling, unnecessary 1 spaces.

File: frontend/src/pages/AccountManager/index.tsx [hunk 3]
```diff
@@ -1,17 +1,226 @@
...
+          />
+          <LabelModal>e-mail: </LabelModal>
+          <InputError
+            name="email"
+            value={formData?.email || ''}
+            onChange={(e) => onChangeData(e, 255)}
+            onBlur={(e) => onChangeData(e, 255)}
+            errorMessage={errors.email}
+          />
+          {!dataEdit?.uid ? (
```
Comment (ReiHashimoto) [inline frontend/src/pages/AccountManager/index.tsx:190]: password form appears on Edit modal.

Screenshot 2023-08-04 at 13 47 04

File: frontend/src/pages/AccountManager/index.tsx [hunk 4]
```diff
@@ -1,17 +1,226 @@
...
+          <InputError
+            name="name"
+            value={formData?.name || ''}
+            onChange={(e) => onChangeData(e, 100)}
+            onBlur={(e) => onChangeData(e, 100)}
+            errorMessage={errors.name}
+          />
+          <LabelModal>Role: </LabelModal>
+          <SelectError
+            value={formData?.role_id || ''}
```
Comment (ReiHashimoto) [inline frontend/src/pages/AccountManager/index.tsx:175]: can you set current role as initial value on Edit modal?

**Review by ReiHashimoto — CHANGES_REQUESTED** [review]

File: frontend/src/pages/AccountManager/index.tsx (hunk 3 above)
Comment (sanglevinh) [reply frontend/src/pages/AccountManager/index.tsx:190]: @ReiHashimoto, This Pull Request is not related to editing the user, so can I disable the edit button and make the necessary changes in a separate Pull Request specifically created for editing the user?

File: frontend/src/pages/AccountManager/index.tsx (hunk 2 above)
Comment (sanglevinh) [reply frontend/src/pages/AccountManager/index.tsx:43]: @ReiHashimoto i fixed it

File: frontend/src/pages/AccountManager/index.tsx (hunk 4 above)
Comment (sanglevinh) [reply frontend/src/pages/AccountManager/index.tsx:174]: @ReiHashimoto This Pull Request is not related to editing the user, so can I disable the edit button and make the necessary changes in a separate Pull Request specifically created for editing the user?

File: frontend/src/pages/AccountManager/index.tsx (hunk 3 above)
Comment (ReiHashimoto) [reply frontend/src/pages/AccountManager/index.tsx:189]: @sanglevinh 
OK, please disable edit button and leave to do comment.

File: frontend/src/pages/AccountManager/index.tsx (hunk 4 above)
Comment (ReiHashimoto) [reply frontend/src/pages/AccountManager/index.tsx:174]: same as
https://github.com/arayabrain/optinist-for-server/pull/96#discussion_r1284155886

File: frontend/src/pages/AccountManager/index.tsx (hunk 2 above)
Comment (ReiHashimoto) [reply frontend/src/pages/AccountManager/index.tsx:43]: @sanglevinh 
It seems to be not fixed.

My request is, 
```
const ModalComponent =
___({  // indent with 3 spaces → 2spaces
_____onSubmitEdit, // indent with 5spaces → 4spaces
...
```

Comment (ReiHashimoto) [reply frontend/src/pages/AccountManager/index.tsx]: @sanglevinh 
sorry for not making my intention clear enough, too.
I found it fixed, thank you.

**Review by ReiHashimoto — APPROVED** [review]

**Pull request merged**